concert/experiments/imaging.py

```diff
@@ -1027,13 +1034,15 @@ async def _get_num_flats(self):
         return 0
 
     async def _set_num_flats(self, n):
-        raise AccessorNotImplementedError
+        if n != 0:
+            raise AccessorNotImplementedError
 
     async def _get_num_projections(self):
         return 1
 
     async def _set_num_projections(self, n):
```

Comment:
I know this is a bit ugly. I will do some generalizations on the grating experiments soon, then this will look better again.

Reply:
Alright